waltid-applications/waltid-cli/src/jvmTest/kotlin/id/walt/cli/verifiers/ExpirationDatePolicyTest.kt

```diff
@@ -45,7 +45,7 @@ class ExpirationDatePolicyTest {
     }
 
     @Test
-    fun `should NOT fail with a valid signed VC and data = jws-payload('vc')`() {
+    fun `should NOT fail with a valid signed VC and data = jws-payload("vc)"`() {
```

Comment:
quote is an illegal char on windows
this is also why it was changed to apostrophe (single-quote)